.travis.yml

```diff
@@ -1,23 +1,18 @@
-dist: xenial
-sudo: required
-services:
-  - docker
```

Comment:
we don't need docker, docker service only let's you run things like `docker pull` or `docker push` it doesn't mean that your test is running on docker